packages/editor/src/services/EditorServices.ts

```diff
@@ -1,9 +1,12 @@
 import { store } from '@xrengine/client-core/src/store'
 import { createState, useState } from '@hookstate/core'
 
-const state = createState({
-  projectName: null as string,
-  sceneName: null as string
+const state = createState<{
+  projectName: string | null
+  sceneName: string | null
+}>({
+  projectName: null,
```

Comment:
we cn do `null! as string` here to avoid manual typing :)

Reply:
`null` is also assigned during runtime in some cases.